student/views.py

```diff
@@ -1,3 +1,23 @@
-# from django.shortcuts import render
+from django.shortcuts import render, HttpResponseRedirect
+from .models import Student
+from .forms import UpdateStudentAccountSettingsForm
 
-# Create your views here.
+
+def account_update_success(request):
+    is_student = Student.is_student(request.user.id)
+    return render(request, 'account_settings_update_success.html', {"is_student": is_student})
+
+
+def update_student_account_settings_view(request):
+    context = {}
+    try:
+        obj = Student.objects.get(user_id=request.user.id)
+    except Student.DoesNotExist:
+        return render(request, "dead_end.html")
+    form = UpdateStudentAccountSettingsForm(request.POST or None, instance=obj)
+    if form.is_valid():
+        form.save()
+        return HttpResponseRedirect("/account_update_success/")
+    context["form"] = form
+    context["is_student"] = Student.is_student(request.user.id)
```

Comment:
what is this for?

Reply:
Same thing I explained to you on 
https://github.com/redhat-beyond/SJMaster/pull/60#discussion_r767267251